cmd/pdc/main.go

```diff
@@ -97,6 +97,22 @@ func main() {
 		os.Exit(1)
 	}
 
+	// Allow overrides of _some_ empty flags with environment variables
+	clusterOverride, ok := os.LookupEnv("GCLOUD_PDC_CLUSTER")
+	if ok && mf.Cluser = nil {
```

Comment:
The nil value for string types is `""`

```suggestion
	if ok && mf.Cluser == "" {
```

Reply:
typo:
```suggestion
	if ok && mf.Cluster == "" {
```